app/views/layouts/_main_nav.html.erb

```diff
@@ -11,22 +15,26 @@
       Sign In/Up
     </a>
   <% end %>
-  <div class="sidebar-navigation-links spec-sidebar-navigation-links">
-    <% nav_links[0..4].each do |link| %>
+  <ul class="default-navigation-links sidebar-navigation-links spec-sidebar-navigation-links">
```

Comment:
This is awesome 😄  I think maybe the "Home" link on line 8 should be inside this list too?